velox/connectors/hive/storage_adapters/abfs/AbfsConfig.h

```diff
@@ -16,44 +16,93 @@
 
 #pragma once
 
+#include <azure/core/credentials/credentials.hpp>
+#include <azure/storage/blobs/blob_client.hpp>
+#include <azure/storage/files/datalake.hpp>
 #include <folly/hash/Hash.h>
 #include <string>
 
+using namespace Azure::Storage::Blobs;
+using namespace Azure::Storage::Files::DataLake;
+
 namespace facebook::velox::config {
 class ConfigBase;
 }
 
 namespace facebook::velox::filesystems {
 
 // This is used to specify the Azurite endpoint in testing.
-static std::string kAzureBlobEndpoint{"fs.azure.blob-endpoint"};
+static constexpr const char* kAzureBlobEndpoint{"fs.azure.blob-endpoint"};
+
+// The authentication mechanism is set in `fs.azure.account.auth.type` (or the
+// account specific variant). The supported values are SharedKey, OAuth and SAS.
+static constexpr const char* kAzureAccountAuthType =
+    "fs.azure.account.auth.type";
+
+static constexpr const char* kAzureAccountKey = "fs.azure.account.key";
+
+static constexpr const char* kAzureSASKey = "fs.azure.sas.fixed.token";
```

Comment:
Let's add documentation for the new config.
`velox/docs/configs.rst`

Reply:
Updated, thanks.